README.md

```diff
@@ -90,6 +91,13 @@ MultiSession.setup do |config|
 
   # Salt used to derive key for GCM encryption. Default value is 'multi session authenticated encrypted cookie'
   config.authenticated_encrypted_cookie_salt = 'my multi session salt value'
+
+  # Specify the strategy by which you are managing credentials in your application
+  # Defaults to :credentials or :secrets depending on what your application is using
+  # 'credentials' (default) - uses Rails encrypted credentials stored in credentials.yml.enc via Rails.application.credentials
+  # 'secrets' - uses Rails secrets specified in secrets.yml via Rails.application.secrets
+  # 'creds' - uses the [Creds](https://github.com/freeletics/creds) gem via Rails.configuration.creds
+  config.credentials_strategy = Rails.application.respond_to?(:credentials) ? 'credentials' : 'secrets'
```

Comment:
No need to have the backward compatibility logic in the README. Just have it as `config.credentials_strategy = 'credentials'`.